src/duckstation-qt/mainwindow.cpp

```diff
@@ -1555,12 +1547,15 @@ void MainWindow::onGameListEntryContextMenuRequested(const QPoint& point)
       menu.addSeparator();
 
       connect(menu.addAction(tr("Select Disc")), &QAction::triggered, this, &MainWindow::onGameListEntryActivated);
+    }
 
-      menu.addSeparator();
+    menu.addSeparator();
 
-      connect(menu.addAction(tr("Exclude From List")), &QAction::triggered,
-              [this, entry]() { getSettingsWindow()->getGameListSettingsWidget()->addExcludedPath(entry->path); });
-    }
+    connect(menu.addAction(tr("Exclude From List")), &QAction::triggered,
```

Comment:
This one's a bit iffy - disc sets don't have a path, so excluding them doesn't really make a lot of sense.

It'd probably "work", but to the user, a name in an excluded path list might be confusing.

Reply:
This was already in both branches of the if/then, I just moved it below to avoid repetition. Are you saying it should not be there for disc sets?

Reply:
Eh, if it's already there, whatever.